src/vision/status_quo/alan_tries_processing_some_files.md

```diff
@@ -0,0 +1,279 @@
+# 😱 Status quo stories: Alan tries processing some files
+
+## 🚧 Warning: Draft status 🚧
+
+This is a draft "status quo" story submitted as part of the brainstorming period. It is derived from real-life experiences of actual Rust users and is meant to reflect some of the challenges that Async Rust programmers face today. 
+
+If you would like to expand on this story, or adjust the answers to the FAQ, feel free to open a PR making edits (but keep in mind that, as they reflect peoples' experiences, status quo stories [cannot be wrong], only inaccurate). Alternatively, you may wish to [add your own status quo story][htvsq]!
+
+## The story
+
+Alan is new to Rust. He wants to build a program that recurses over all the files in a directory (and its subdirectories), reads each file, and produces some fingerprint of the file.
+
+Since so much blocking I/O is involved, he chooses async in order to process many files concurrently.
+
+### Recursion
+Alan starts by writing a recursive function that can call some operation on each regular file in a directory and recurse on each subdirectory.
```

Comment:
One thing I wondered here: is Alan using tokio? How did he decide to use tokio?

Reply:
Ah, good point - I've added an `### Async` section to provide a little bit about ~~my~~ Alan's route to using Tokio.